mos/compute/tasks.py

```diff
@@ -21,10 +21,14 @@ def model_run(model_id, model_name, caller_id):
             backend_host, 
             backend_port))
 
-        token = interface.get_user_token(
-            os.getenv('MOS_ADMIN_USR'),
-            os.getenv('MOS_ADMIN_PWD'),
-        )
+        if os.getenv('MOS_BACKEND_TOKEN') == None:
```

Comment:
Please use
```
if not os.getenv('MOS_BACKEND_TOKEN'):
```
Let's also update the readme under "Environment Variables":
* MOS_ADMIN_USR and MOS_ADMIN_PWD, or MOS_BACKEND_TOKEN